plugins/slick.customtooltip.js

```diff
@@ -260,35 +274,51 @@
 
         // do the same calculation/reposition with top/bottom (default is top of the cell or in other word starting from the cell going down)
         if (position === 'top' || (position === 'auto' && calculatedTooltipHeight > calculateAvailableSpaceTop(cellElm))) {
-          newPositionTop = cellPosition.top + (_gridOptions.rowHeight || 0) + (_options.offsetTopBottom || 0);
+          newPositionTop = cellPosition.top + (_gridOptions.rowHeight || 0) + (_cellTooltipOptions.offsetTopBottom || 0);
           _tooltipElm.classList.remove('arrow-down');
           _tooltipElm.classList.add('arrow-up');
         } else {
           _tooltipElm.classList.add('arrow-down');
           _tooltipElm.classList.remove('arrow-up');
         }
 
-        // reposition the editor over the cell (90% of the time this will end up using a position on the "right" of the cell)
+        // reposition the tooltip over the cell (90% of the time this will end up using a position on the "right" of the cell)
```

Comment:
change to `align-left` and `align-right` since it's not really a cell left/right position